eox_core/settings/test.py

```diff
@@ -22,8 +22,10 @@ def plugin_settings(settings):  # pylint: disable=function-redefined
     settings.EOX_CORE_PRE_ENROLLMENT_BACKEND = "eox_core.edxapp_wrapper.backends.pre_enrollment_h_v1"
     settings.EOX_CORE_COURSEKEY_BACKEND = "eox_core.edxapp_wrapper.backends.coursekey_h_v1"
     settings.EOX_CORE_CERTIFICATES_BACKEND = "eox_core.edxapp_wrapper.backends.certificates_h_v1_test"
+    settings.EOX_CORE_CONFIGURATION_HELPER_BACKEND = "eox_core.edxapp_wrapper.backends.configuration_helpers_h_v1_test"
```

Comment:
Are all the new added settings with the  suffix "_h_v1" compatible with hawthorn? otherwise the suffix should be "_i_v1" or something of that sort to indicate that this is valid/compatible for ironwood.